UnitsNet/GeneratedCode/Quantities/AmplitudeRatio.g.cs

```diff
@@ -111,14 +111,14 @@ public AmplitudeRatio(double decibelvolts)
 
         #region Properties
 
-		/// <summary>
-		///     The <see cref="QuantityType" /> of this quantity.
-		/// </summary>
+        /// <summary>
```

Comment:
Not sure what happened to the indentation here. The new one looks right to me though

Reply:
Might have been a sneaky tab char, or merge conflict when merging in the master changes to your branch.